docs/conf.py

```diff
@@ -92,12 +92,12 @@
     "github_repo": "exodus-lambda",
     "github_button": False,
     "github_banner": True,
-    "description": "AWS Lambda functions for Red Hat's Content Delivery Network",
+    "description": "AWS Lambda functions for Red Hat's Content Delivery Network",  # noqa
```

Comment:
Could this please be avoided?  I think this line is actually fine, so rather than adding noqa comments I'd rather fix the settings of whichever linter prompted you to add this.

I see in pyproject.toml right now we have a line length of 79 defined there, but I think it was a mistake, we're usually having a limit of 100.  Would you mind doing one of either:

- find/update/add the appropriate config file in this repo so your linter accepts line limit of 100, or
- just remove these noqa from the commit